godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -63,7 +62,7 @@
 import java.util.List;
 import java.util.Set;
 
-public class GodotGooglePlayBilling extends GodotPlugin implements PurchasesUpdatedListener, BillingClientStateListener, PriceChangeConfirmationListener {
```

Comment:
`PriceChangeConfirmationListener` is [deprecated](https://developer.android.com/reference/com/android/billingclient/api/PriceChangeConfirmationListener)